java/ehcache/Portfile

```diff
@@ -1,53 +1,54 @@
 PortSystem 1.0
```

Comment:
Now that you fixed the spacing: check other ports which contain some special commented-out settings in the first line and copy-paste that one here.

Reply:
Hi, I've added the same.